idseq_pipeline/commands/common.py

```diff
@@ -640,7 +640,8 @@ def run_and_log_work(log_params, target_outputs, lazy_run, func_name,
     global OUTPUT_VERSIONS
     global run_and_log_mutex
 
-    write_to_log("========== %s ==========", log_params.get("title"))
+    msg = "========== %s ==========" % log_params.get("title")
```

Comment:
Maybe not to be addressed on this PR, but why are we not using logger directly?
There are multiple benefits: you can use handlers to output to multiple places, you do not have to process the string (i.e. '%') and you can be more flexible with the log levels.